array-manipulation-2 (1).tar/array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,8 @@
+/* exported take */
+function take(array, count){
+  const result=[];
```

Comment:
there's a quick way to address this:

Use the `Math` object, specifically the `min` method, and pass in the `array.length` and `count` arguments. This will determine the smaller of the two numbers. You can then loop over the smaller number instead of the `count` variable so that you won't push `undefined` to indices that don't exist on the original array.